packages/labs/gen-utils/tsconfig.json

```diff
@@ -1,7 +1,7 @@
 {
   "compilerOptions": {
-    "target": "es2020",
-    "module": "es2022",
+    "target": "ES2020",
```

Comment:
why capitalized?

Reply:
I was getting errors in vs code without changing this.